src/components/services/ActionQueue.js

```diff
@@ -52,7 +52,8 @@ export const ActionQueue = () => {
 
           const { program, state } = queue[programId]
           const { nextState } = await executeActionProgram(account, program, state).catch((error: Error) => {
-            console.error('Action Program Exception:', error.message)
+            console.warn(new Error('Action Program Exception: ' + error.message))
+            console.error(error)
```

Comment:
Again with the redbox. We should only use `console.error` when the program crashes, not for generic error logging. Use `warn` to get a yellow box, or log to go to the output stream.